src/client.rs

```diff
@@ -317,27 +319,37 @@ impl fmt::Debug for RequestBuilder {
 }
 
 /// A Response to a submitted `Request`.
+#[derive(Debug)]
```

Comment:
The old version of debug has been implemented on `Decoder` since the `Decoder::Plaintext` variant is now wrapping the hyper request. So deriving debug here makes sense since all `Request` is doing is wrapping a `Decoder` now. [See changes on line 418](https://github.com/seanmonstar/reqwest/pull/61/files#diff-31bbf71c54bca98a0ae3d40a327af940R418)